src/Traits/Body/ChecksForWithBody.php

```diff
@@ -19,10 +19,10 @@ trait ChecksForWithBody
      */
     public function bootChecksForWithBody(PendingRequest $pendingRequest): void
```

Comment:
The interface was renamed to HasBody, which it now checks for.

```suggestion
    public function bootChecksForHasBody(PendingRequest $pendingRequest): void
```